sdks/java/core/build.gradle

```diff
@@ -117,6 +132,113 @@ task packageTests(type: Jar) {
   classifier = "tests"
 }
 
+class ValidatesRunnerConfig {
+  // Task name prefix
+  String testPrefix
```

Comment:
Merge testPrefix, runnerName and configuration all into one field so that we just standardize how everything is defined like in examples/java/build.gradle for the precommits.